spec/rails_helper.rb

```diff
@@ -1,4 +1,6 @@
 require "spec_helper"
+require  'coveralls'
+Coveralls.wear_merged!("rails")
```

Comment:
these two lines need to be at the top of this file

Reply:
done